src/platform/tls_openssl.c

```diff
@@ -555,6 +570,155 @@ CxPlatTlsClientHelloCallback(
     return SSL_CLIENT_HELLO_SUCCESS;
 }
 
+int
+CxPlatTlsOnClientSessionTicketReceived(
+    _In_ SSL *Ssl,
+    _In_ SSL_SESSION *Session
+    )
+{
+    CXPLAT_TLS* TlsContext = SSL_get_app_data(Ssl);
+
+    BIO* Bio = BIO_new(BIO_s_mem());
+    if (Bio) {
+        if (PEM_write_bio_SSL_SESSION(Bio, Session) == 1) {
```

Comment:
i2d_SSL_SESSION would be more efficient as it skips bas64 wrapping but it may not matter. The PEM format may be easier to debug.